apps/dot-voting/contracts/test/mocks/DotVotingMock.sol

```diff
@@ -5,7 +5,7 @@ import "../../DotVoting.sol";
 
 contract DotVotingMock is DotVoting {
     // _isValuePct public wrapper
-    function isValuePct(uint256 _value, uint256 _total, uint256 _pct) external pure returns (bool) {
```

Comment:
Why remove this? It shouldn't throw a compiler error since the internal function is also pure.

Reply:
@Quazia this is a question for you

Reply:
Or maybe this is a question for you, @ottodevs 😄